kubemarine/procedures/check_iaas.py

```diff
@@ -288,6 +288,30 @@ def system_distributive(cluster):
         
         tc.success(results=", ".join(detected_supported_os))
 
+def check_kernel_version(cluster):
+    """
+    This method compares the linux kernel version with the bad version
+    """
+    with TestCase(cluster.context['testsuite'], '015', "Software", "Kernel version") as tc:
+        bad_results = []
+        bad_kernel_ubuntu = ['5.4.0-132-generic']
+        bad_kernel_centos = []
+        group = cluster.nodes['all']
+        result_group = group.sudo('uname -r && cat /etc/os-release', warn=True)
+        for host, results in result_group.items():
+            result = results.stdout.split('\n')
+            if 'Ubuntu' in result[1]:
```

Comment:
There is a `KubernetesCluster.get_os_family_for_node` or `NodeGroup.get_nodes_os` method to get nodes' OS families.

Reply:
done